src/types/tweetdeckTypes.ts

```diff
@@ -527,6 +528,8 @@ interface User {
   _profileBannerURL: string;
   prototype: this;
   getExpandedURL(): string;
+  setFollowing(following: boolean): void;
+  prettyFollowersCountInTweetAction?(): string;
```

Comment:
Let's mark those methods as optional since they can potentially be undefined if the follow action is disabled/something went wrong.

Reply:
I have done it in 835b5c2.